.ci/pytorch/test.sh

```diff
@@ -1000,8 +1000,25 @@ test_docs_test() {
 
 test_executorch() {
   # Test torchgen generated code for Executorch.
-  echo "Testing Executorch op registration"
+  echo "Testing ExecuTorch op registration"
   "$BUILD_BIN_DIR"/test_edge_op_registration
```

Comment:
Does this line run the actual test? If so, shouldn't this be run after torchvision/audio are installed?

Reply:
Hmm, true.  The test passed, so I guess it doesn't require vision/audio here.  It's better to run it after installing vision and audio anyway, so let me move it down